src/aat/java/uk/gov/hmcts/reform/em/annotation/functional/RectangleScenarios.java

```diff
@@ -0,0 +1,387 @@
+                .body("annotationId", equalTo(annotationId))
+                .log().all();
+    }
+
+    @Test
+    public void shouldReturn404WhenGetRectangleNotFoundById() {
+        final String rectangleId = UUID.randomUUID().toString();
+        request
+                .get("/api/rectangles/" + rectangleId)
+                .then()
+                .statusCode(404)
+                .log().all();
+    }
+
+    @Test
+    public void shouldReturn401WhenGetRectangleById() {
+        final String rectangleId = UUID.randomUUID().toString();
+        unAuthenticatedRequest
+                .get("/api/rectangles/" + rectangleId)
+                .then()
+                .statusCode(401)
+                .log().all();
+    }
+
+    @Test
+    public void shouldReturn200WhenGetAllRectangles() {
+        final String newAnnotationSetId = createAnnotationSet();
+        final String annotationId = createAnnotation(newAnnotationSetId);
+        final String rectangleId = UUID.randomUUID().toString();
+        createRectangle(annotationId, rectangleId);
+
+        request
+                .get("/api/rectangles")
+                .then()
+                .statusCode(200)
+                .body("size()", Matchers.greaterThanOrEqualTo(1))
+                .log().all();
+    }
+
+    @Test
+    public void shouldReturn401WhenGetAllRectangles() {
+        unAuthenticatedRequest
+                .get("/api/rectangles")
+                .then()
+                .statusCode(401)
+                .log().all();
+    }
+
+    @Test
+    public void shouldReturn200WhenUpdateRectangle() {
+        final String newAnnotationSetId = createAnnotationSet();
+        final String annotationId = createAnnotation(newAnnotationSetId);
+        final String rectangleId = UUID.randomUUID().toString();
+        final ValidatableResponse response = createRectangle(annotationId, rectangleId);
+        final JSONObject rectangle = extractJSONObjectFromResponse(response);
+        rectangle.put("x", 3f);
+        rectangle.put("y", 4f);
+
+        request
+                .body(rectangle.toString())
+                .put("/api/rectangles")
+                .then()
+                .statusCode(200)
+                .body("id", equalTo(rectangleId))
+                .body("x", equalTo(3f))
+                .body("y", equalTo(4f))
+                .body("width", equalTo(10f))
+                .body("height", equalTo(11f))
+                .body("annotationId", equalTo(annotationId))
+                .log().all();
+    }
+
+    @Test
+    public void shouldReturn500WhenUpdateRectangle() {
+        final String rectangleId = UUID.randomUUID().toString();
+        final String annotationId = UUID.randomUUID().toString();
+        final JSONObject rectangle = createRectanglePayload(annotationId, rectangleId);
+        request
+                .body(rectangle.toString())
+                .put("/api/rectangles")
+                .then()
+                .statusCode(500)
```

Comment:
again, it's not clear why

Reply:
its same as above issue, raised a bug for this issue as well.